lib/id3c/cli/command/etl/kit.py

```diff
@@ -1,539 +0,0 @@
-"""
-Process both enrollment and manifest documents to extract kit related data
-into the relational warehouse.
-
-The kit etl is completely dependent on the enrollments etl and the manifest etl.
-It does not create encounters or samples, but must find them within the
-relational warehouse.
-"""
-import click
-import logging
-from psycopg2 import sql
-from datetime import datetime, timezone
-from typing import Any, Optional, Tuple
-from id3c.cli.command import with_database_session
-from id3c.db import find_identifier
-from id3c.db.session import DatabaseSession
-from id3c.db.datatypes import Json
-from id3c.db.types import KitRecord, SampleRecord
```

Comment:
KitRecord and SampleRecord can be removed from id3c.db.types too, as this appears to be their sole callsite.